programs/bubblegum/program/src/lib.rs

```diff
@@ -1348,6 +1619,94 @@ pub mod bubblegum {
         )
     }
 
+    pub fn update_metadata<'info>(
+        ctx: Context<'_, '_, '_, 'info, UpdateMetadata<'info>>,
+        root: [u8; 32],
+        nonce: u64,
+        index: u32,
+        current_metadata: Option<MetadataArgs>,
+        update_args: UpdateArgs,
+    ) -> Result<()> {
+        assert_signed_by_tree_delegate(&ctx.accounts.tree_authority, &ctx.accounts.tree_delegate)?;
```

Comment:
Talked about this a bit offline, but I think we should talk about allowing Tree Delegate to update Metadata.

Reply:
Yeah had a couple more ideas about this, discussing offline.  We can also change this before we deploy it as I am still planning to remove `update_metadata_collection_nft` before we deploy this.

Reply:
We resolved this online. TLDR my comment here was mostly due to a misunderstanding of cNFT UAs. I assumed there was a UA field in the metadata when in fact it's the Tree creator. So adding permissions to the Tree delegate isn't authority creep.